class/class/personne.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace class personne
```

Comment:
Чтобы ваша программа заработала нужно либо оформить ваш класс как библиотеку классов (что требовалось в задании ООП-1), либо разместить его в том же пространстве имен, что и основная программа.
```suggestion
namespace @class
```